doc/sflow/sflow_hld.md

```diff
@@ -85,13 +88,22 @@ The newly introduced sflow container consists of:
 * sflowmgrd : updates APP DB sFlow tables based on config updates
 
 The swss container is enhanced to add the following component:
-* sfloworch : which subscribes to the APP DB and acts as southbound interface to SAI for programming the SAI_SAMPLEPACKET sessions.
-* copporch : Copporch gets the genetlink family name and multicast group from copp.json file, programs the SAI genetlink attributes and associates it with trap group present for sflow in copp.json
+1. sfloworch : which subscribes to the APP DB and acts as southbound interface to SAI for programming:
+i. sFlow sessions: SAI_SAMPLEPACKET APIs
+ii. MOD: SAI_TAM APIs
+
+2. copporch : Copporch gets the genetlink family name and multicast group from copp.json file, programs the SAI genetlink attributes and associates it with trap group present for sflow in copp.json.
+For MOD, SflowOrch creates the following:
+i.  Hostif of type SAI_HOSTIF_TYPE_GENETLINK and associate with GENETLINK family "NET_DM" and mcgroup "events".

```

Comment:
can we add sample copp_cfg.j2 capturing new trap group and UDT ?

Reply:
This User Defined Trap and Trap group are created by sflowOrch dynamically based on MOD CLI. It is NOT processed by copporch. Even the TAM APIs are invoked by sFlowOrch-- This is the implementation of Edgecore and they said they will revisit if there is a pushback from community and eBay+Aviz are ok with this proposal.